Fix: Typos in Gateway get started (#7722)
Kong/docs.konghq.com: app/_src/gateway/get-started/key-authentication.md

With {{site.base_gateway}} controlling authentication, requests won't reach upstream services unless the client has successfully
authenticated. This means upstream services process pre-authorized requests, freeing them from the 

> cost of authentication, which is a saving in compute time *and* development effort.

Reviewer: ```suggestion
cost of authentication, which reduces computing time *and* development effort.
```
Hmm, I wonder if this could be rephrased to omit "savings" in general since that feels more "marketing-y" speak.
